jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -242,7 +249,7 @@ pipeline {
                                                 string(name: 'AGENT_LABEL', value: AGENT_X64)
                                             ]
 
-                                        createTestResultsMessage(
+                                        env.ARTIFACT_URL_X64_RPM_INTEG_TEST_RESULT = createTestResultsMessage(
```

Comment:
does this need the explicit `env`?

Reply:
The variable will be used in the post -> always block. I think we need to make it explicit to make it accessible.